### DISCUSSION TITLE
Can I avoid the first Enter when creating a note?

### DISCUSSION THREAD

**Discussion by jpsala** [SilentVoid13/Templater#1063] [Help] [answered]
Sorry for the title :)
When I create a templater note: I use my hotkey, templater asks me for a template, so I select the template and hit enter, all good, my note is created, but... instead of going to the cursor, it waits for me to name/accept the filename, which I already named inside the script, I would love templater to go to tp.file.cursor instead, can I do that?
I'm sharing my template just in case it's useful for this
```
<%*
// const ID = await tp.system.prompt("Ticket ID:");
//const description = (await tp.system.prompt("Description:")).replace('/', ' ').replace('?', ' ').replace(':', ' ').replace('&', ' ').replace(',', ' ').replace('.', ' ');
let filename = tp.file.title
const ticket = await tp.system.prompt("Ticket:")
const {id, description, url} = await tp.user.jsForTicketTemplate(tp, ticket)
await tp.file.move("Tickets/" + id);
-%>
# <% id %> ^id
## <% description %> ^title
><% url %> ^url`$=dv.current()?.tags?.includes('#active')?"<i style='color:red;font-size: 80%'><br/> Active</i>":""`

>[!info]+ Description
><% tp.file.cursor() %>

### <% tp.date.now('YYYY-MM-DD') %> ^work
- 
---

### Doubts
-








---
>[!metadata]- Metadata
>Tags:: <% `#` %>active, <% `#` %>ticket
>creation-date:: <% tp.date.now("YYYY-MM-DD ddd HH:mm:ss") %>
>ID:: "<% id %>"
>description:: "<% description  %>"
>url:: "<% url  %>"
>ticket-status:: In Progress
```

**Comment by Zachatoo** [comment]
You need to turn on `Show tab title bar` in Appearance settings.

**Reply by jpsala** [reply]
Thank you for your answer!
But actually, I have that setting enabled, if I disable it, it gets worst, because obsidian asks me the title in a popup, with that setting enabled, when I create the note, obsidian waits at the top of the note for me to change the title and press enter or click outside of it, which it's not a bid deal, but If I can avoid it, then it would be better for me, make sense? English is not my native language so maybe I'm not been clear :(
What I want to know is if there's a way to:
`

Press my hotkey and be asked by templater to choose one template
Choose my template and once the note is created, I want templater to goes to my <% tp.file.cursor() %>
I'm showing my template again just in case I'm doing something wrong:


<%*
// const ID = await tp.system.prompt("Ticket ID:");
//const description = (await tp.system.prompt("Description:")).replace('/', ' ').replace('?', ' ').replace(':', ' ').replace('&', ' ').replace(',', ' ').replace('.', ' ');
let filename = tp.file.title
const ticket = await tp.system.prompt("Ticket:")
const {id, description, url} = await tp.user.jsForTicketTemplate(tp, ticket)
await tp.file.move("Tickets/" + id);
-%>
# <% id %> ^id
## <% description %> ^title
><% url %> ^url`$=dv.current()?.file?.tags?.includes('#active')?"<i style='color:red;font-size: 80%'><br/> Active</i>":""`

>[!info]+ Description
<% tp.file.cursor() %>

### <% tp.date.now('YYYY-MM-DD') %> ^work
- 
---

### Doubts
-








---
>[!metadata]- Metadata
>Tags:: <% `#` %>active, <% `#` %>ticket
>creation-date:: <% tp.date.now("YYYY-MM-DD ddd HH:mm:ss") %>
>ID:: "<% id %>"
>description:: "<% description  %>"
>url:: "<% url  %>"
>ticket-status:: In Progress




Linked mentions
0
`

**Reply by Zachatoo** [reply]
Unfortunately I don't know a way around this. I believe this was likely not an issue before, but with Obsidian 1.0 and the new tab title bar this is an issue.

You could make a bug report on the issues page, though I'm not sure where the maintainer has gone, so it may not be fixed anytime soon unfortunately.

**Comment by jpsala** [comment]
Thank you Zach!

**Comment by Zachatoo** [comment, marked as answer]
I'm adding a "Jump to note content after creating note" setting, when enabled, will jump your cursor to the note content without having to tab into it.

<img width="888" height="776" alt="Screenshot 2026-06-05 at 10 05 48 PM" src="https://github.com/user-attachments/assets/5a5b7074-66b9-4f63-b266-d272141d2d7e" />

This is in the Templater 2.22.0 release, which is only available on Obsidian 1.13.0+
